Dockerfile

```diff
@@ -14,12 +14,13 @@ ARG S3CMD_VERSION="2.0.2"
 ARG SOPS_VERSION="3.6.1"
 ARG TERRAFORM_VERSION="0.12.19"
 ARG YQ_VERSION="3.2.1"
+ARG JQ_VERSION="1.6"
```

Comment:
Minor nit, they are alphabetically sorted right now. Please move before you merge.

Reply:
Done